js/everything.js

```diff
@@ -509,11 +594,52 @@ $(function() {
 
     errorMessage.init();
 
+    videoMetaData = window.sessionStorage.getItem("videoMetaData");
```

Comment:
We need a function for this, we're duplicating code for retrieval and parsing the JSON